src/ts/helpers/hours.ts

```diff
@@ -8,16 +8,20 @@ const subtypes = [
   'off-road',
 ];
 
+export function isHour(offer): boolean {
+  if (!offer) return false;
+  return _.includes(subtypes, offer.subtype);
```

Comment:
this is not specific for hourly rentals. something like this perhaps? `_.get(offer, 'subtypes.hourly')`